docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1257,102 +1183,18 @@ find_in_set(str, strlist)
 - **str**: String expression to find in strlist.
 - **strlist**: A string list is a string composed of substrings separated by , characters.
 
-## Binary String Functions
```

Comment:
in https://datafusion.apache.org/user-guide/sql/scalar_functions_new.html#binary-string-functions